.github/workflows/build-and-push-image-development.yml

```diff
@@ -58,3 +58,20 @@ jobs:
               --resource-group ${{ secrets.DEVELOPMENT_ACA_RESOURCE_GROUP }} \
               --image ${{ secrets.DEVELOPMENT_ACR_URL }}/complete-app:${{ steps.prepare-tags.outputs.deploy-version }} \
               --output none
+
+      - name: Azure login with ACA Worker credentials
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.DEVELOPMENT_AZURE_ACA_WORKER_CREDENTIALS }}
+
+      - name: Update Azure Container Apps Worker Revision
+        uses: azure/CLI@v1
+        with:
+          azcliversion: 2.40.0
+          inlineScript: |
+            az config set extension.use_dynamic_install=yes_without_prompt
+            az containerapp update \
+              --name ${{ secrets.DEVELOPMENT_ACA_CONTAINERAPP_NAME }}-worker \
+              --resource-group ${{ secrets.DEVELOPMENT_ACA_RESOURCE_GROUP }} \
+              --image ${{ secrets.DEVELOPMENT_ACR_URL }}/complete-app:${{ steps.prepare-tags.outputs.deploy-version }} \
```

Comment:
Given that `set-output` is [deprecated](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/), should we take this opportunity to change this?

Reply:
I've created a Trello card for us to update these, we'll update them across all environments at once 👍